.gitignore

```diff
@@ -1,7 +1,9 @@
 # Please keep this list in ASCII sort order
 .idea
+step240-2020.iml
```

Comment:
Please move items to be in [ASCII order](http://support.ecisolutions.com/doc-ddms/help/reportsmenu/ascii_sort_order_chart.htm). (For the most part, it is alphabetical)